tests/accounts/test_views.py

```diff
@@ -205,16 +207,127 @@ def test_first_name(self):
 
 class UserViewTestCase(TestCase):
     def setUp(self):
-        self.user = User.objects.create(
-            pennid=1, username="test1", first_name="first1", last_name="last1"
+        self.user = get_user_model().objects.create_user(
+            pennid=1,
+            username="student",
+            password="secret",
+            first_name="First",
+            last_name="Last",
+            email="test@test.com",
         )
+
+        Major.objects.create(name="Test Active Major", is_active=True, degree_type="BACHELORS")
+        Major.objects.create(name="Test Active Major 2", degree_type="PHD", is_active=True)
+        Major.objects.create(name="Test Active Major 3", degree_type="PROFESSIONAL", is_active=True)
+
+        School.objects.create(name="Test School")
+        School.objects.create(name="Test School 2")
+
+        Student.objects.create(user=self.user)
+        self.user.student.major.add(Major.objects.get(name="Test Active Major"))
+        self.user.student.major.add(Major.objects.get(name="Test Active Major 2"))
+        self.user.student.school.add(School.objects.get(name="Test School"))
+        self.user.student.graduation_year = 2024
+        self.serializer = StudentSerializer(self.user.student)
+
         self.client = APIClient()
         self.serializer = UserSerializer(self.user)
 
     def test_get_object(self):
         self.client.force_authenticate(user=self.user)
         response = self.client.get(reverse("accounts:me"))
         self.assertEqual(json.loads(response.content), self.serializer.data)
+        self.assertEqual(response.status_code, 200)
+        """print(json.dumps(json.loads(response.content), indent=4))"""
```

Comment:
Don't forget to remove all the comments/print statements

Reply:
fixed